target/scripts/startup/setup-stack.sh

```diff
@@ -444,48 +444,48 @@ function _setup_postfix_postscreen
 
 function _setup_postfix_sizelimits
 {
-  _log 'trace' "Configuring postfix message size limit to ${POSTFIX_MESSAGE_SIZE_LIMIT}"
+  _log 'trace' "Configuring Postfix message size limit to '${POSTFIX_MESSAGE_SIZE_LIMIT}'"
   postconf -e "message_size_limit = ${POSTFIX_MESSAGE_SIZE_LIMIT}"
 
-  _log 'trace' "Configuring postfix mailbox size limit to ${POSTFIX_MAILBOX_SIZE_LIMIT}"
+  _log 'trace' "Configuring Postfix mailbox size limit to '${POSTFIX_MAILBOX_SIZE_LIMIT}'"
   postconf -e "mailbox_size_limit = ${POSTFIX_MAILBOX_SIZE_LIMIT}"
 
-  _log 'trace' "Configuring postfix virtual mailbox size limit to ${POSTFIX_MAILBOX_SIZE_LIMIT}"
+  _log 'trace' "Configuring Postfix virtual mailbox size limit to '${POSTFIX_MAILBOX_SIZE_LIMIT}'"
   postconf -e "virtual_mailbox_limit = ${POSTFIX_MAILBOX_SIZE_LIMIT}"
 }
 
 function _setup_clamav_sizelimit
 {
-  _log 'trace' "Configuring ClamAV message scan size limit to ${CLAMAV_MESSAGE_SIZE_LIMIT}"
+  _log 'trace' "Setting ClamAV message scan size limit to '${CLAMAV_MESSAGE_SIZE_LIMIT}'"
   sedfile -i "s/^MaxFileSize.*/MaxFileSize ${CLAMAV_MESSAGE_SIZE_LIMIT}/" /etc/clamav/clamd.conf
 }
 
 function _setup_postfix_smtputf8
 {
-  _log 'trace' "Configuring postfix smtputf8 support (disable)"
+  _log 'trace' "Disabling Postfix's smtputf8 support"
   postconf -e "smtputf8_enable = no"
 }
 
 function _setup_spoof_protection
 {
-  _log 'trace' "Configuring Spoof Protection"
+  _log 'trace' 'Configuring spoof protection'
   sed -i \
     's|smtpd_sender_restrictions =|smtpd_sender_restrictions = reject_authenticated_sender_login_mismatch,|' \
     /etc/postfix/main.cf
 
   if [[ ${ENABLE_LDAP} -eq 1 ]]
   then
     if [[ -z ${LDAP_QUERY_FILTER_SENDERS} ]]; then
```

Comment:
Something you might be interested in :sweat_smile: 

```suggestion
    if [[ -z ${LDAP_QUERY_FILTER_SENDERS} ]]
    then
```

Reply:
😆